cypress/integration/ResetPassword/index.spec.ts

```diff
@@ -0,0 +1,25 @@
+describe('Reset password page', () => {
+  describe('given the email', () => {
+    it('renders the success message', () => {
+      cy.interceptApi('/me', {
+        statusCode: 401,
+        fixture: 'requests/unauthorizedUserProfile',
+      }).as('getUserProfile');
+
+      cy.interceptApi('/passwords', {
+        fixture: 'requests/resetPassword',
+      }).as('resetPassword');
+
+      cy.visit('/password-reset');
+
+      cy.findByLabelText('Email').type('user@mail.com');
+      cy.findByRole('button', { name: /Send Recovery Email/ }).click();
+
+      cy.wait(['@getUserProfile', '@resetPassword']);
+
+      cy.findByText(
+        "We've email you instructions to reset your password."
```

Comment:
```suggestion
        "We've emailed you instructions to reset your password."
```

You did it well, but the Figma Design is wrong xD

Reply:
I also missed that part 🙈